generate-docs/action.yml

```diff
@@ -0,0 +1,96 @@
+name: "generate-docs"
+description: "Generate documentation for the project"
+
+inputs:
+  version-name:
+    description: "Version name to use in the documentation"
+    required: true
+  github-access-token:
+    description: "If the repository is private, specify a token."
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        # Fetch with depth=0 to push to gh-pages
+        fetch-depth: 0
+
+    - name: Install MkDocs
+      run: |
+        python3 -m pip install --upgrade pip setuptools wheel
+        pip3 install -r ${GITHUB_ACTION_PATH}/mkdocs-requirements.txt
+      shell: bash
+
+    # TODO: remove after v1.2.0 or v2.0.0 is released
+    - name: (tmp) Install latest mike
+      run: |
+        pip install git+https://github.com/jimporter/mike
+      shell: bash
+
+    - name: Remove non-document files
+      run: |
+        find . -type f \
+            -not -path './.git/*' \
+            -not -path './.github/*' \
+            -not -path './mkdocs.yml' \
+            -not -path './mkdocs.yaml' \
+            -not -name '*.md' \
+            -not -name '*.svg' \
+            -not -name '*.png' \
+            -not -name '*.css' \
+          | xargs -I {} rm "{}"
+      shell: bash
+
+    - name: Build MkDocs
+      run: |
+        mkdocs-yaml=$(find . -type f -path './mkdocs.yaml' -or -path './mkdocs.yml')
+        mkdocs build -f "$mkdocs-yaml" --verbose --clean
+      shell: bash
+
+    - name: Set git config
+      run: |
+        git config --local --unset-all http.https://github.com/.extraheader || true
+        git config --global url.https://x-access-token:${{ inputs.github-access-token }}@github.com.insteadOf 'https://github.com'
+        git config --global user.name github-actions
+        git config --global user.email github-actions@github.com
+      shell: bash
+
+    - name: Deploy
+      run: |
+        git fetch
+        mkdocs-yaml=$(find . -type f -path './mkdocs.yaml' -or -path './mkdocs.yml')
+        mike deploy -F "$mkdocs-yaml" --push --rebase ${{ inputs.version-name }}
+      shell: bash
+
+    - name: Create comment body
+      if: ${{ github.event.pull_request }}
+      id: create-comment-body
+      run: |
+        mkdocs-yaml=$(find . -type f -path './mkdocs.yaml' -or -path './mkdocs.yml')
+        site_url=$(grep -oP "site_url:\s+\K(.*)" "$mkdocs-yaml")
+        base_url="${site_url}/pr-${{ github.event.pull_request.number }}/"
+        ${GITHUB_ACTION_PATH}/create-doc-comment.sh "${base_url}" "origin/${{ github.event.pull_request.base.ref }}"
```

Comment:
Please change the comment in `create-doc-comment.sh` like this.

`# Create comment body for generate-docs action`

Reply:
daf3528